src/engine/blkstore/blkstore.hpp

```diff
@@ -325,6 +325,9 @@ class BlkStore {
     BlkAllocStatus alloc_contiguous_blk(const uint32_t size, blk_alloc_hints& hints, BlkId* const out_blkid) {
         // Allocate a block from the device manager
         assert(size % m_pagesz == 0);
+        static uint32_t max_suppoted_size = m_pagesz * std::numeric_limits< blk_count_t >::max();
```

Comment:
is there any reason it needs to be `static`? it can be `uint32_t const max_supported_size =...`, right?